src/components/mobile-menu-container/index.tsx

```diff
@@ -40,10 +40,11 @@ const MOBILE_MENU_MOTION = {
  */
 const MobileAuthenticationControls = () => {
 	const { asPath } = useRouter()
-	const { showAuthenticatedUI, showUnauthenticatedUI, signIn, signOut, user } =
```

Comment:
I chose to remove `showAuthenticatedUI` booleans because they were only used in one place and the logic can be inferred already from values that are passed from `useAuthentication` hook.